### ISSUE TITLE
tracks with no start annotated, missing start 1 in graphics of Track

### ISSUE THREAD

**Issue by cdshaffer** [SEA-PHAGES/starterator#19]
I have run  across examples where start 1 is not marked on the graphical tracks. examples include the following phams (with protein member in case pham number changes)

pham no.   member:          note:
21767        UPIE_87           5 members in pham, 2 tracks, all annotate start 1
21710        Dave_Draft_1   5 members in pham, 1 track, all annotate start 1

**Comment by cdshaffer** [comment]
This problem may also be causing crashes, and may be related to genomes that start with the first gene starting at base 1. 

I have found that in database version 71 that pham 6635 is crashing starterator. it has the following genes all of which begin at base 1:

Members of Pham 6635

```
Gene            Phage   Cluster Length
Grum1_Draft_1   Grum1   A   177 bp
Ollie_Draft_1   Ollie   A   177 bp
Sabia_Draft_1   Sabia   A   177 bp
Zetzy_Draft_1   Zetzy   A   177 bp
```

**Comment by cdshaffer** [comment]
closer examination shows that the first three bases of these genes is not a valid start codon but instead the first three bases are TGC. this leads to erratic behavior and crashing in starterator. This is clearly an edge case, not sure why it is being annotated this way. 

get phage sequence and run with auto annotation in DNA Master- confirmed that DNA Master is annotating bases 1 - 177 as a gene. Running the sequence of the whole phage on the genmark web page at NCBI I can see that genemark is indicating it is a partial gene. It looks like DNA Master interprets this as an intact gene which then flows into phamerator. It also looks like phagesdb is translating it incorrectly and giving the frame3 translation (i.e. giving the translation of bases 3 - 176) and not the frame 1 translation (bases 1 - 174).

this is clearly a corner case. Might be good to fix at the level of either auto-annotation or curation of draft sequences into phamerator. could add special code in starterator to more gracefully handle the issue instead of crashing but does not appear to be very high priority.

**Comment by cdshaffer** [comment]
I have added code to check all annotated starts during pham creation. If a gene's annotation is not a valid start codon in (ATG, GTG, or TTG) then the gene is removed from the pham. There are several phams where all genes are of this type so the program still fails on these but 2 phams there is a mix of valid and invalid genes. Added warning to pham report to warn readers if genes have been removed.
